compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -1013,12 +1013,18 @@ impl<'hir> Map<'hir> {
                 ItemKind::Use(path, _) => path.span,
                 _ => named_span(item.span, item.ident, item.kind.generics()),
             },
+            Node::Variant(variant) => match variant.data {
```

Comment:
Let's return the same thing in all cases.  It will be more consistent with how we treat tuple structs where we stop at the first parenthesis.